pkgs/servers/adguardhome/default.nix

```diff
@@ -16,8 +13,8 @@ stdenv.mkDerivation rec {
   meta = with lib; {
     homepage = "https://github.com/AdguardTeam/AdGuardHome";
     description = "Network-wide ads & trackers blocking DNS server";
-    platforms = platforms.linux;
-    maintainers = with maintainers; [ numkem ];
+    platforms = platforms.all;
+    maintainers = with maintainers; [ numkem iagoq ];
     license = licenses.gpl3;
```

Comment:
gpl3Plus or gpl3Only?

Reply:
It's gpl3Only apparently. No mentions of later versions in their repository.